semantic_conventions/trace/http.yaml

```diff
@@ -22,6 +22,16 @@ groups:
           is often, but not always, present as the [Content-Length](https://www.rfc-editor.org/rfc/rfc9110.html#field.content-length)
           header. For requests using transport encoding, this should be the compressed size.
         examples: 3495
+      - id: request.size
+        type: int
+        brief: >
+          Total size in bytes of the request (body and headers).
```

Comment:
compressed or uncompressed or "what went over the wire"?

Reply:
OpenCensus used to have a spec for this: https://github.com/census-instrumentation/opencensus-specs/blob/master/stats/HTTP.md#measures

That was uncompressed and did not include headers.